### ISSUE TITLE
Feature request: UI should show which improvements are possible on given terrain

### ISSUE THREAD

**Issue by ghost** [yairm210/Unciv#6879]
When you use non-automated workers, it is very important to know which improvements could be constructed on given tile. However, this is not obvious. E.g. I was recently surprised that a pasture cannot be built on a *desert* hill tile with sheep, despite Civilopedia apparently never saying this. Bananas also don't seem to support plantations, even if Civilopedia says otherwise.

When you tap on a tile, a small box is currently shown with its terrain, yielded resources and current improvement, if any. **Request**: also show *potential* improvements on this tile, e.g. just as icons to conserve space (tapping the icons should open relevant Civilopedia page). Improvements that are possible only after unlocking some technology could be shown with a greyed-out icon, to avoid confusion.

**Comment by xlenstra** [comment]
> a pasture cannot be built on a desert hill tile with sheep

Which version are you on? We've recently had some bugs with what improvements could be build where. The improvement you describe should just be possible, so it could also be that.

Other than that, if you select the 'build improvement' action of a worker, all improvements that can be build are shown, though this doesn't include improvements locked behind a tech.

**Comment by anton-koerbchen** [comment]
> Which version are you on? 

Git says it is commit 1f3890e2a.

> Other than that, if you select the 'build improvement' action of a worker, all improvements that can be build are shown, though this doesn't include improvements locked behind a tech.

After you have spent three turns moving a worker into position. And then you waste another couple of turns moving the worker elsewhere, after having found out that the desired improvement is not actually possible.

Current situation makes it more difficult than necessary to plan worker usage. If someone doesn't use autoworkers, that's likely because the user wants full control over what his workers are doing. Including ability to plan, "ok, I create a farm there, then the worker moves two tiles over here to create a plantation". Without hints what could be built where, it becomes difficult to plan unless you are so familiar with the game as to remember all the improvements by heart.

**Comment by anton-koerbchen** [comment]
I also opened issue #6893 about plantations on banana tiles, but that got closed as won't fix. However, it highlights the poor UI/documentation that doesn't give me any hints why I couldn't build the plantation in the first place.

**Comment by SomeTroglodyte** [comment]
Hey, this is a 6X game! 'Explore, Expand, Exploit, Exterminate', + 'Groxk the rules' + 'Dixcover the xode'

**Comment by SomeTroglodyte** [comment]
This is solved to an extent by #6947, right? Or should we leave it open to check whether any Civilopedia tweaks are yet to be done?
